src/views/AddExpenses.js

```diff
@@ -20,66 +26,104 @@ const AddExpenses = ({ addExpense, setMonthToShow }) => {
     setTitle('');
     setDate('');
     setAmount('');
+    setWasTitleValidated(false);
```

Comment:
the name started me off into the wrong direction. i thought we do our own validation, but we are only enable the bootstrap markup for the browser validation.

Reply:
Yep! I could change the name of the state we use because we are actually not validating through `bootstrap` but only passing a class to render if the browser is returning any empty fields.